mappings/servers/imineplex/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+    "id": "imineplex",
+    "name": "iMinePlex",
+    "description": "Play without lag",
+    "addresses": [
+        "iMinePlex.iR"
+    ],
+    "primaryAddress": "Play.iMinePlex.iR",
+    "minecraftVersions": [
+        "1.8.*",
+        "1.9.*",
+        "1.20.*"
```

Comment:
we don't support 1.9 or 1.20.
This list should only include versions which **both** your server and CB support